webui/react/src/utils/color.ts

```diff
@@ -255,3 +256,23 @@ export const maxColorDistance = (rgba0: RgbaColor, rgba1: RgbaColor): number =>
 export const rgbDistance = (rgba0: RgbaColor, rgba1: RgbaColor): number => {
   return pointDistance([rgba0.r, rgba0.g, rgba0.b], [rgba1.r, rgba1.g, rgba1.b]);
 };
+
+export const cssVarToThemeVar = (cssVar: string): ThemeVariable => {
+  const regEx = /var\(--theme-(.*?)\)/;
+  const themeVarStringArray = cssVar.match(regEx)?.[1].split('-');
+  const themeVariable = themeVarStringArray?.[0].concat(
+    themeVarStringArray
+      .slice(1)
+      .map((propertyName: string) => propertyName.replaceAll('interactive', 'ix'))
+      .map((propertyName: string) => propertyName.charAt(0).toUpperCase() + propertyName.slice(1))
```

Comment:
nit: can we do this in one go rather than mapping over the array again? Also, look into `upperFirst` from lodash -- if the chain feels a bit unwieldy, consider expressing this using `pipe`

assuming the following imports
```ts
import { pipe, upperFirst } form 'lodash'
```
```suggestion
      .map((propertyName: string) => pipe(
        propertyName.replaceAll('interactive', 'ix'),
        upperFirst
       ))
```

Reply:
@ashtonG I did look into this but did want to merge since I know people are waiting for this to merge to complete other work. 

It does not look like `pipe` does not exist in lodash? Did you mean `flow` or `compose`? Or is there a `pipe` method I am missing somewhere?